<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFinder</class>
 <widget class="QWidget" name="TextFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Search</string>
       </property>
       <property name="buddy">
        <cstring>findText</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findText"/>
     </item>
     <item>
      <widget class="QCheckBox" name="wholeWordsCheck">
       <property name="toolTip">
        <string>Search for whole words only</string>
       </property>
       <property name="text">
        <string>&amp;Whole Words</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseSensitiveCheck">
       <property name="toolTip">
        <string>Make search case sensitive</string>
       </property>
       <property name="text">
        <string>&amp;Case Sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="toolTip">
        <string>Open another file</string>
       </property>
       <property name="text">
        <string>&amp;Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findText</tabstop>
  <tabstop>wholeWordsCheck</tabstop>
  <tabstop>caseSensitiveCheck</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
